kvrocks.conf

```diff
@@ -727,6 +727,25 @@ rocksdb.cache_index_and_filter_blocks yes
 # default snappy
 rocksdb.compression snappy
 
+# Specify the compression level to use.
```

Comment:
I thought it would be better to add: `It trades compression speed and ratio, might be useful when tuning for disk space`.

Or we attach https://github.com/facebook/rocksdb/wiki/Space-Tuning link to guide users.

Reply:
Sure, add both comment and link in the updated PR :)